src/woeusb

```diff
@@ -2004,6 +2009,19 @@ util_check_function_parameters_quantity(){
 	return 0
 }; declare -fr util_check_function_parameters_quantity
 
+##Check every file in source directory for files bigger than max fat32 file size (~4GB)
+check_for_big_files(){
+	local source_fs_mountpoint="${1}"
```

Comment:
* *I* pedantically require variables to be [set read-only if it is not likely to be modified](https://github.com/slacka/WoeUSB/blob/master/CONTRIBUTING.md#defensive-bash-programming)